avro/schemas/redeem_request.avsc

```diff
@@ -0,0 +1,27 @@
+{
+    "namespace": "brave.cbp",
+    "type": "record",
+    "doc": "Top level request containing the data to be processed, as well as any top level metadata for this message.",
+    "name": "RedeemRequestSet",
+    "fields" : [
+        {"name": "request_id", "type": "string"},
+        {
+            "name": "data",
+            "type": {
+                "type": "array",
+                "items": {
+                    "namespace": "brave.cbp",
+                    "type": "record",
+                    "name": "RedeemRequest",
+                    "fields": [
+                        {"name": "associated_data", "type": "bytes", "doc": "contains METADATA"},
+                        {"name": "issuer_type", "type": "string"},
+                        {"name": "token_preimage", "type": "string"},
+                        {"name": "signature_source", "type": "string"},
```

Comment:
is this the same as `binding` in `VerificationRequest`?

Reply:
I think yes. `binding` is a much better name. I'll change to that.